### ISSUE TITLE
Gradle Troubleshooting chapter

### ISSUE THREAD

**Issue by oleg-nenashev** [gradle/cookbook#2]
It would be great to start the "Gradle Troubleshooting" chapter that references external tools that are currently not included in the Main Gradle documentation. It could include the following:

- Gradle Doctor - https://github.com/runningcode/gradle-doctor
- OpenTelemetry plugin for Gradle - https://github.com/craigatk/opentelemetry-gradle-plugin
- Gradle Profiler - https://github.com/gradle/gradle-profiler
- Of course, Gradle Build Scan :) 
- ... add your favorite tools to the comments!

## Steps

- [ ] Create the chapter
- [ ] Add tools incrementally

## Note

Currently we cannot include repositories as documentation sources, so short summary pages for key use-cases are preferable

**Comment by anurag-verma-india** [comment]
>## Note
>Currently we cannot include repositories as documentation sources, so short summary pages for key use-cases are preferable

Since Gradle doctor has [github pages](https://runningcode.github.io/gradle-doctor/) for documentation so can we use these right?

**Comment by oleg-nenashev** [comment]
Yes, @anurag-verma-india . A good starter would be a list of the references to the tools with a short overview/summary. No need to duplicate content when it is already present somewhere
